lib/livebook_web/channels/js_dynamic_channel.ex

```diff
@@ -0,0 +1,68 @@
+defmodule LivebookWeb.JSDynamicChannel do
+  use Phoenix.Channel
+
+  @impl true
+  def join("js_dynamic", %{}, socket) do
+    {:ok, assign(socket, ref_with_pid: %{}, ref_with_count: %{})}
+  end
+
+  @impl true
+  def handle_in("connect", %{"session_token" => session_token, "ref" => ref}, socket) do
+    {:ok, data} = Phoenix.Token.verify(LivebookWeb.Endpoint, "js dynamic", session_token)
```

Comment:
We really use the token just to pass the pid, so will change this to `term_to_binary |> base*`.